articles/migrate-to-shared.dd

```diff
@@ -16,14 +16,15 @@ $(HEADERNAV_TOC)
 
 $(H2 $(LNAME2 performance, Performance of TLS variables))
 
-        $(P Yes, reading or writing TLS variables will be slower
-        than for classic global variables. It'll be about 3
-        instructions rather than one.
-        But on Linux, at least, TLS will be slightly faster
-        than accessing classic globals using PIC (position
-        independent code) compiler code generation settings.
-        So it's hard to see that this would be an unacceptable
-        problem. But let's presume it is. What can we do about it?
+        $(P Reading or writing TLS variables is slower than for classic
+        global variables. The exact difference depends, a.o., on compiler
```

Comment:
What does `a.o.` mean? "Among other"? Can you expand the abbreviation not to confuse readers?

Reply:
yes